Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -21,11 +21,18 @@ public ElevatorMaintainerCommand(ElevatorSubsystem elevator, PIDFactory pf) {
     @Override
     public void initialize() {
         log.info("Initializing");
+        if (!elevator.isCalibrated()) {
+            log.warn("ELEVATOR UNCALIBRATED - THIS COMMAND WILL NOT DO ANYTHING!");
+        }
     }
 
     @Override
     public void execute() {
-        double power = pid.calculate(elevator.getTargetHeight(), elevator.getCurrentHeight());
-        elevator.setPower(power);
+        if (elevator.isCalibrated()) {
+            double power = pid.calculate(elevator.getTargetHeight(), elevator.getCurrentHeightInInches());
```

Comment:
Why are we using a software PID loop rather than that which is built into the Talons?

Reply:
there's a PID thing in the Talons!?

Reply:
Simplicity & ease of understanding for new programmers, but we can upgrade this to use the TalonSRX onboard PID in the future.